stories/media/video.stories.js

```diff
@@ -3,15 +3,29 @@ import { withInfo } from '@storybook/addon-info';
 
 import { Video } from '@sparkpost/matchbox';
 import Analytics from '@sparkpost/matchbox-media/videos/Analytics.webm';
+import Analytics2 from '@sparkpost/matchbox-media/videos/Analytics.mp4';
+
+import { Box } from '@sparkpost/matchbox';
 
 export default {
   title: 'media|Video',
 };
 
 export const BasicVideo = withInfo({ propTables: [Video] })(() => (
-  <Video>
-    <Video.Source src={Analytics} type="video/webm" />
-  </Video>
+  <Box display="flex">
+    <Box>
+      <Box as="h4">webm</Box>
+      <Video>
+        <Video.Source src={Analytics} type="video/webm" />
```

Comment:
You can add two sources here under the same `<Video>` 😬

Reply:
Oh duh 🤦